src/app/item-page/bitstreams/request-a-copy/bitstream-request-a-copy-page.component.ts

```diff
@@ -209,4 +248,112 @@ export class BitstreamRequestACopyPageComponent implements OnInit, OnDestroy {
   getBitstreamLink() {
     return [getBitstreamDownloadRoute(this.bitstream)];
   }
+
+  /**
+   * execute the captcha function for v2 invisible
+   */
+  executeRecaptcha() {
+    this.googleRecaptchaService.executeRecaptcha();
+  }
+
+  /**
+   * Register an email address
+   */
+  register(tokenV2?) {
```

Comment:
This appears to be copied from the registration form (`register-email-form.component.ts`), but the name is wrong for a request a copy form.  We are not registering an email address on this form.  Maybe this needs to be renamed to "requestCopy" or something similar? 

All variables/methods named "register" need to be renamed as this form has nothing to do with registration